### DISCUSSION TITLE
Select grain boundary between two grains

### DISCUSSION THREAD

**Discussion by dbugas** [mtex-toolbox/mtex#1437] [Ask Anything]
Is it possible to only analyze properties of a grain boundary only between two grains from ebsd data of a polycrystal, such as length, misorientation, and so on? Also, if the grain boundary between the two grains is broken up by a different grain, can the different segments be distinguished? 

Thank you

**Comment by kilir** [comment]
Hi,
try
```
mtexdata titanium
grains = ebsd.calcGrains
sp = [36 42 ]
gb = grains.boundary;
gbs =  gb(all(gb.grainId == [sp],2));

plot(grains([sp]))
hold on
plot(gbs,'linecolor','fuchsia')
hold off
```

Hope this helps.
Rüdiger
